modules/oe_content_person/oe_content_person.module

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file
+ * The OpenEuropa Person Content module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_BASE_FORM_ID_alter for the node form.
+ */
+function oe_content_person_form_node_form_alter(array &$form, FormStateInterface $form_state) {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $form_state->getBuildInfo()['callback_object']->getEntity();
+
+  if ($node->bundle() !== 'oe_person') {
+    return;
+  }
+
+  $form['oe_departments']['#states'] = [
+    'visible' => [
+      ':input[name="oe_person_type"]' => [
+        'value' => 'eu',
+      ],
+    ],
+  ];
+
+  $form['oe_person_organisation']['#states'] = [
+    'visible' => [
+      ':input[name="oe_person_type"]' => [
+        'value' => 'non_eu',
+      ],
+    ],
+  ];
```

Comment:
This could be a good occasion to add a method to this utility class, ans use it here:
```
\Drupal\oe_content\Utility\ContentFormUtilities
```

Reply:
done.

I notice there is no coverage for the class, shall we add a Kernel ?